Makefile

```diff
@@ -7,7 +7,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsupdate:
-	docker-compose run --rm cli bash -c "composer update && composer show -Df json > versions.json"
+	docker-compose run --rm cli bash -c "git config --global --add safe.directory /data/vendor/silinternational/yii2-codeception && composer update && composer show -D > versions.json"
```

Comment:
It seems odd to see `git config` calls when merely trying to update composer dependencies. What necessitated this?

Reply:
I was getting something similar to this (below). Apparently, there''s a new security feature that docker containers mess with somehow.

https://stackoverflow.com/questions/71901632/fatal-error-unsafe-repository-home-repon-is-owned-by-someone-else